packages/client/ui/react-ui/src/components/hosted/styles.ts

```diff
@@ -48,4 +49,11 @@ export const useStyles = createUseStyles({
         color: ({ paragraphColor }: CustomStylingProps) => paragraphColor,
         margin: "0",
     },
-});
+};
+
+export const useStyles: (
+    data?:
+        | CustomStylingProps & {
+              theme?: any;
+          }
+) => Classes<"crossmintButton" | "crossmintImg" | "crossmintParagraph" | "@global"> = createUseStyles(styles);
```

Comment:
this is a difficult one, but the problem is that previously the return type was inferrered - and then when this was compiled it throws error 
```
The inferred type of 'useStyles' cannot be named without a reference to '.pnpm/jss@10.10.0/node_modules/jss'. This is likely not portable. A type annotation is necessary
```

so, what im adding here is a defined type - its not pretty, but it works....